portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,9 +35,22 @@
 public class DataServlet extends HttpServlet {
 
   private List<String> comments = new ArrayList<>();
+  // Declares instance of Datastore Service
```

Comment:
nit: You probably don't need a comment here, because it's self explanatory from the code that you are creating a new DatastoreService entity! Comments are more useful when they explain why/how you're doing something, rather than what you are doing.

Reply:
Ack!